base/markdown/render/terminal/render.jl

```diff
@@ -61,6 +61,36 @@ function term(io::IO, md::Code, columns)
     end
 end
 
+function term(io::IO, md::Table, columns)
+    col_widths = reduce(max, map(cell -> map(ansi_length, cell), md.rows))
+
+    for (n, row) in enumerate(md.rows)
+        for (i, h) in enumerate(row)
+            a = typeof(md.align) == Symbol ? md.align : md.align[i]
+            print_align(io, h, col_widths[i], a)
+            print(io, " ")
+        end
+        println(io, "")
+
+        if n == 1
+            for w in col_widths
+                print(io, ("-" ^ w) * " ")
+            end
+            println(io, "")
+        end
+    end
+end
+
+function ansi_length(md::Vector{Any})
```

Comment:
Apparently I missed this issue. Just to comment, FYI, that `length(terminline(md))` didn't work because it doesn't skip the ansi codes, unfortunately plaininline had a side effect, so I modified it to just return the string. Not sure how to get rid of this call elegantly (since it needs to work recursively).